src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -1466,6 +1467,82 @@ namespace OpenLoco::Ui::WindowManager
         invalidateWidget(window->type, window->number, widgetIndex);
     }
 
+    static bool isStepperGroup(Window& w, WidgetIndex_t index, WidgetType buttonType)
+    {
+        const auto& widgets = w.widgets;
+
+        if (widgets[index].type != WidgetType::textbox && widgets[index].type != WidgetType::wt_3)
+            return false;
+
+        if (widgets[index + 1].type != buttonType)
+            return false;
+
+        if (widgets[index + 2].type != buttonType)
+            return false;
+
+        return true;
+    }
+
+    static std::optional<WidgetIndex_t> getStepperGroupWidgetIndex(Window& w, WidgetIndex_t startIndex)
+    {
+        // We only iterate 3 times as we might be at the the textbox or one of the buttons.
+        for (WidgetIndex_t index = 0; index < 3; index++)
+        {
+            const auto reverseIndex = startIndex - index;
+            if (reverseIndex < 0)
+                break;
+
+            if (isStepperGroup(w, reverseIndex, WidgetType::toolbarTab))
+                return reverseIndex;
+
+            if (isStepperGroup(w, reverseIndex, WidgetType::button))
+                return reverseIndex;
+        }
+
+        return std::nullopt;
+    }
+
+    // Allow mouse wheel scrolling to manipulate stepper widgets and tool sizes
+    static bool stepperWheelInput(Window& w, WidgetIndex_t widgetIndex, int32_t wheel)
+    {
+        const auto stepperGroupIndex = getStepperGroupWidgetIndex(w, widgetIndex);
+        if (!stepperGroupIndex.has_value())
+            return false;
```

Comment:
missing `{ }` here; the other if statements below have it so may as well add it here?  (and maybe in the above function too?